CONTRIBUTING.md

```diff
@@ -73,18 +73,9 @@ To add or update a go dependency:
   make test PKG=./kv TESTS='^TestFoo' TESTTIMEOUT=10s
   ```
 
-  When you're ready to commit, do just that with a succinct title and
-  informative message. For example,
-
-  ```bash
-  $ git commit
-  > 'update CONTRIBUTING.md
-  >
-  > Added details on running specific tests via `make`, and
-  > the CircleCI-equivalent test suite.
-  >
-  > Fixed some formatting.'
-  ```
+  When you're ready to commit, be sure to write a Good Commit Message™. Consult
+  https://github.com/erlang/otp/wiki/Writing-good-commit-messages if you're
+  not sure what constitutes a Good Commit Message™.
```

Comment:
Done.

On Thu, Aug 18, 2016 at 12:52 PM, Bram Gruneir <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/cockroachdb/cockroach/pull/8651#discussion_r75345401>:
>
> > @@ -73,18 +73,9 @@ To add or update a go dependency:
> >    make test PKG=./kv TESTS='^TestFoo' TESTTIMEOUT=10s
> >    ```
> >
> > -  When you're ready to commit, do just that with a succinct title and
> > -  informative message. For example,
> > -
> > -  ```bash
> > -  $ git commit
> > -  > 'update CONTRIBUTING.md
> > -  >
> > -  > Added details on running specific tests via `make`, and
> > -  > the CircleCI-equivalent test suite.
> > -  >
> > -  > Fixed some formatting.'
> > -  ```
> > +  When you're ready to commit, be sure to write a Good Commit Message™. Consult
> > +  https://github.com/erlang/otp/wiki/Writing-good-commit-messages if you're
> > +  not sure what constitutes a Good Commit Message™.
>
> Be sure to add how to handle ones that involve more than one package, or
> packages that are not at the top level (i.e. stopper).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/8651/files/d2d811af3dbdba868cb9109fe0f1b3958f67ee29#r75345401>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPHqJDog-YXv_p25TRVgIiXuU159Rks5qhI2_gaJpZM4JnsR->
> .
>